misc-scripts/xref_mapping/xref_config2sql.pl

```diff
@@ -175,6 +177,8 @@
     my $source_section = sprintf( "source %s", $source_name );
     $source_section =~ s/\s$//;
 
+    next if ($preparse && $species_id == 7742 && $source_name eq 'RefSeq_peptide::MULTI-vertebrate');
```

Comment:
Suggest to put a comment in the code to make explicit 7742 is related to vertebrates.
This should turn out useful also because "vertebrates" is no species.